### PR TITLE
Added a new choice message and welcome message

### PR DESCRIPTION
**Pull request by sidtr0**

### CODE REVIEW

File: src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt [hunk 1]
```diff
@@ -100,6 +101,7 @@ object Messages {
         "Hello, %name%, I don't suppose you're a tester? Mods like to see a tester around, it gives them something to aim at.",
         "%name% has a mind that ticks like a clock and, like a clock, it regularly goes cuckoo!",
         "%name% gives their compliments to the mods, and begs them to keep their abnormally large noses out of other people's business."
+        "Have you ever heard the tragedy of %name% the Wise? I hought not. It's not a story the mods would tell you."
```
Comment (itsHobbes) [inline src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt]: You have a typo here! "thought" not "hought"

Comment (sidtr0) [reply src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt]: Fixed!

File: src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt [hunk 2]
```diff
@@ -10,6 +10,7 @@ object Messages {
         "Perhaps... %choice%.",
         "%choice% sounds good to me.",
         "If it were up to me, I'd go with %choice%"
```
Comment (itsHobbes) [inline src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt]: Missing a comma at the end of the line

File: src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt [hunk 3]
```diff
@@ -100,6 +101,7 @@ object Messages {
         "Hello, %name%, I don't suppose you're a tester? Mods like to see a tester around, it gives them something to aim at.",
         "%name% has a mind that ticks like a clock and, like a clock, it regularly goes cuckoo!",
         "%name% gives their compliments to the mods, and begs them to keep their abnormally large noses out of other people's business."
```
Comment (itsHobbes) [inline src/main/kotlin/me/markhc/hangoutbot/locale/Messages.kt]: Missing a comma at the end of the line

**Review by itsHobbes — CHANGES_REQUESTED** [review]

**Review by itsHobbes — APPROVED** [review]

**Pull request merged**